src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
I'm torn on this one tbh. I feel like there's a band-aid aspect that would just feel better if we made the change now and were done with it, but to have done that right I think we would have needed to let people know we were deprecating it a while back. Either that, or be prepared to help existing plugins migrate.

Either option is fine, but given the time constraints, I think we'd be better to add it to core with a deprecation notice (in the dev version at least) and remove it in the next major.